app/crt0/crt0_boot_trap.s

```diff
@@ -46,7 +46,7 @@ csr_init:
   csrw mtvec, t0    # Write the value in t0 to the mtvec CSR
 
   # Enable software and external interrupts. Set msie, meie  fields Disable the mtie timer interrupt 
-  li t0, 0x808
+  li t0, 0x888
```

Comment:
Not sure we want to enable this by default
I think 0x808 is better

Reply:
yes I know, we talked about thay. I just wanted you to check the `100` with out touching other things :-)